arbitrary-file-write/file-create/README.md

```diff
@@ -49,4 +49,21 @@ asdas
 
 [we]! can have symbols and garbage at the end
 <00><ff> binary also ok here
-```
\ No newline at end of file
+```
+### Shell Profiles
+#### Login Shells
+`--login` flag. 
+* `/etc/profile` executed for all users
```

Comment:
Does this file not already exist? Should this not be moved over to the `file-overwrite` category.
Also we need examples how it reacts to script errors, binary data, ... 
We need examples how to execute an actual command.